index.js

```diff
@@ -74,6 +86,7 @@ function fetch(baton) {
     }).then(res => {
       baton.size = Number(res.headers['content-length'])
       baton.data = res.body
+      baton.updateColor()
```

Comment:
This would be removed in favor of a global function called in the promise chain later.